### ISSUE TITLE
booming blade 2nd damage add description

### ISSUE THREAD

**Issue by InfernoIV** [kakaroto/Beyond20#813]
**Describe the bug**
If you roll a weapon attack using booming blade, it shows 2 lines of damage:

- Thunder (Booming Blade) Damage: x
- Thunder (Booming Blade) Damage: y

Please add an description on which damage is which.
In the spell tab, it is described as:
_"2d8 Damage
3d8 Damage
(Moving Target)"_

**To Reproduce**
1. Go to character sheet
2. Click on weapon, to see details
3. Click on Beyond20 logo behind "Booming Blade"
4. See the rolls in chat, but without explanation

**Expected behavior**
I expect text to be added "(Moving Target)", as described in the spells.

**Screenshots**

![image](https://user-images.githubusercontent.com/5473121/120866632-ad656d00-c590-11eb-8a90-598ebba8d580.png)

![image](https://user-images.githubusercontent.com/5473121/120866797-ee5d8180-c590-11eb-99ca-5edbd47e4aef.png)

![image](https://user-images.githubusercontent.com/5473121/120866616-a3436e80-c590-11eb-9918-123c31496181.png)

**Browser Info (please complete the following information):**
 - Browser: Edge
 - Foundry Version: 0.8.6
 - Beyond20 Version: 2.4.3

**Additional context**
Link to the character sheet: ddb.ac/characters/26710121/6Ylpa2

**Comment by Aeristoka** [comment]
The issue here begins very swiftly to become the length of the message, at this point it would be:
Thunder (Booming Blade) (Moving Target) Damage:

Which might look ok (but not really at all) on FoundryVTT, and horrifying on Roll20 (huge user base).

In my opinion, this is one of those "The Player and the GM need to know the spell parts, and when they apply", not something for Beyond20 to do.

@kakaroto , your thoughts?

**Comment by kakaroto** [comment]
Yeah, the reason I don't include that extra text is because it **can** be pretty long, making it unreadable, especially on Roll20, but even in Foundry or Discord, it becomes too much. While the "Moving target" is not so bad, there are other spells where you have a much more descriptive text, and already including the damage type + the spell it came from is making it large.

That being said, I'm not dismissing this entirely because it is a valid concern. Especially since the damage can be different and you may not want to hover on each to know what it applies to, **but** the damages should be respecting the same order, so you would know be able to know that the second thunder damage is the one for the moving target because that's how it appears in D&D Beyond.

Keeping the issue open, but with the `unsure` tag, to be evaluated later. 
Thanks.

**Comment by InfernoIV** [comment]
I don't know what the possibilities are, but I have 2 ideas:

1. Add a tooltip on the damage roll describing the pre-requisites of the damage (a DC that must be passed, target has to move, etc.). This will make it look clean in Foundry, and if needed you can hover the mouse over the damage to see the pre-requisites. Not sure if that is possible though. 

2. Add the option "Damage long description" to Beyond20? In that way, (new) players can enable it to post the whole damage effect description (of every item, since it's a global option), and disable it when it's no longer needed.   

Just some ideas, please let me know you thoughts.
